test/tla/cli-integration-tests.md

```diff
@@ -3978,6 +3978,31 @@ The Apalache server is running on port 8888. Press Ctrl-C to stop.
 ...
 ```
 
+### server mode: error is nicely reported when port is already in use
+
+Start socat in the background to occupy the port, save its pid and wait a second to let binding happen
+```sh
+$ socat TCP-L:8888,fork,reuseaddr - & echo $! > pid.pid && sleep 1
+```
+
+Try to start the Apalache server on the occupied port,
+redirect its output to the file,
+save its exit code,
+strip logger prompt 
+and exit with status code of the server
+```sh
+$ apalache-mc server --port=8888 1>out 2>out; ext=$? && cat out | sed -E 's/E@.*$//' && exit $ext
+...
+Error while starting Apalache server: Failed to bind to address 0.0.0.0/0.0.0.0:8888: Address already in use
```

Comment:
Actually, this string has a suffix, added by logger, that contains a time, that's why I need all this crazy stuff when running the server, to cut it off

Reply:
Its a beautifully written black box test, given the constraints of our wonky system. Kudos to working out all it fits together!